help/mobile-notifications.md

```diff
@@ -167,6 +167,18 @@ To enable push notifications for your organization:
 
 {end_tabs}
 
+#### Warning banners
+
+To make sure that you are aware when your server does not have mobile
+notifications enabled, you will see a warning banner if:
+
+- Your server's access to mobile push notifications is about to end. This banner
+  is first shown to server administrators, and then to all users.
+- Notifications are not enabled on your server.
+
+These banners can be snoozed temporarily, or permanently silenced in
+notification settings.
```

Comment:
Maybe "in the mobile app's notification settings panel."? I don't want someone being confused as to why that doesn't appear in the web app's notification settings, which is irrelevant to this banner.

Reply:
This is just a placeholder; we'll do  #28707 imminently to actually add the instructions.